lib/generators/templates/setup/bin_setup.rb

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+require "fileutils"
+require "open3"
+
+# path to your application root.
+APP_ROOT = File.expand_path("..", __dir__)
+
+def system!(*args)
+  system(*args, exception: true)
+end
+
+FileUtils.chdir APP_ROOT do
+  if ENV["CI"].nil?
+    puts "\n== Installing Homebrew Bundle from Brewfile =="
+    system! "brew bundle"
+
+    puts "\n== Starting postgresql@15 =="
+    system! "brew services restart postgresql@15"
+
+    puts "\n== Starting redis =="
+    system! "brew services restart redis"
+
+    _, _, status = Open3.capture3("which asdf")
+    if status.success?
+      puts "== Installing tool dependecies via asdf =="
+      system! "asdf plugin-add nodejs https://github.com/asdf-vm/asdf-nodejs"
+      system! "asdf plugin-add ruby https://github.com/asdf-vm/asdf-ruby"
+      system! "asdf plugin-add yarn https://github.com/twuni/asdf-yarn"
```

Comment:
There's a case to be made we should conditionally install `nodejs` and `yarn`, since we can't guarantee when this generator will be run. 
```suggestion
      system! "asdf plugin-add nodejs https://github.com/asdf-vm/asdf-nodejs" if File.exist?("package.json")
      system! "asdf plugin-add ruby https://github.com/asdf-vm/asdf-ruby"
      system! "asdf plugin-add yarn https://github.com/twuni/asdf-yarn" if File.exist?("package.json")
```